src/pages/offer-details/OfferDetails.constants.ts

```diff
@@ -58,8 +60,9 @@ export const activeButtonActions = ({
     buttons.push({
       label: 'common.labels.sendMessage',
       buttonProps: {
-        disabled: true,
-        variant: ButtonVariantEnum.Tonal
+        disabled: myOffer,
```

Comment:
I guess, `myOffer ` prop should be renamed to i`sMyOffer` as it is a boolean

Reply:
myOffer was renamed